test.py

```diff
@@ -424,8 +442,18 @@ def build_test_list(self) -> List[str]:
     def pattern(self) -> str:
         assert False
 
-    async def add_test(self, shortname) -> None:
-        test = PythonTest(self.next_id((shortname, self.suite_key)), shortname, self)
+    def cases(self, shortname: str) -> List[str]:
+        # Find test cases in pytest file (without importing), so they can be invoked separately
+        pytest_file = str(self.suite_path / (shortname + ".py"))
+        with open(pytest_file, "rt") as file:
```

Comment:
also, this syntax with test cases won't allow for general [pytest testIDs](https://docs.pytest.org/en/7.1.x/example/parametrize.html#different-options-for-test-ids), like class names and specific test function parameter values. If we just pass the `-k` command line option to pytest as is, this won't be a problem.

Reply:
Done in https://github.com/scylladb/scylladb/commit/ff48059e2cb97764a9f44a1567153892698c18ab